hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1128,3 +1129,35 @@ def hpat_pandas_series_le_impl(self, other):
         return hpat_pandas_series_le_impl
 
     raise TypingError('{} The object must be a pandas.series and argument must be a number. Given: {} and other: {}'.format(_func_name, self, other))
+
+
+@overload_method(SeriesType, 'abs')
+def hpat_pandas_series_append(self):
+    """
+    Pandas Series method :meth:`pandas.Series.abs` implementation.
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_abs1
+
+    Parameters
+    -----------
+    self: :obj:`pandas.Series`
+          input series
+
+    Returns
+    -------
+    :obj:`pandas.Series`
+         returns :obj:`pandas.Series` containing the absolute value of elements
+    """
+
+    _func_name = 'Method abs().'
+
+    if not isinstance(self, SeriesType):
```

Comment:
I think we need additional check like this. As I see this method can work with numeric data only but Series can contain, for example, strings (not at this time but in general).
In pandas.abs() documentation we have `This function only applies to elements that are all numeric.`. I think we need to raise exception if underling data is not numeric.

Reply:
@shssf In 1135 line "def hpat_pandas_series_append(self):", but it must be   "def hpat_pandas_series_abs(self):".